lib/internal/perf/timerify.js

```diff
@@ -53,6 +53,7 @@ function processComplete(name, start, args, histogram) {
     entry[n] = args[n];
 
   enqueue(entry);
+  return this;
```

Comment:
It's weird to utilize `this` to pass the result...

Reply:
yes it it, because `FunctionPrototypeBind` bind result as `this`